### PR TITLE
fix: [FX-3050] Use davinci by parts

### PR DESCRIPTION
**Pull request by denieler**
[FX-3050]

### Description

Use davinci by parts.
Fixed some READMEs to enourage people to use davinci by parts in their projects

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

[FX-3050]: https://toptal-core.atlassian.net/browse/FX-3050?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by denieler** [comment]
@toptal-anvil ping reviewers

File: upload-source-maps/action.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@ runs:
...
         fi
 
-        if [[ $(yarn davinci help) == *"sentry-upload-source-maps"* ]]
+        if [[ $(yarn davinci-engine help) == *"sentry-upload-source-maps"* ]]
         then
-          yarn davinci engine sentry-upload-source-maps
+          yarn davinci-engine sentry-upload-source-maps
         else
-          echo "Your current davinci version is not supporting uploading source maps to Sentry. Please update davinci package version in your app."
+          echo "Your current @toptal/davinci-engine version is not supporting uploading source maps to Sentry. Please update @toptal/davinci-engine package version in your app."
```
Comment (LashaJini) [inline upload-source-maps/action.yml:37]: How about

```suggestion
          echo "Your current @toptal/davinci-engine version does not support uploading source maps to Sentry. Please update @toptal/davinci-engine package version in your app."
```

File: README.md [hunk 2]
```diff
@@ -24,8 +24,7 @@ jobs:
     name: Check:Danger
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout davinci GHAs
-        uses: toptal/davinci-github-actions/yarn-install@v3.0.1
+      - uses: toptal/davinci-github-actions/yarn-install@v3.0.1
```
Comment (dmaklygin) [inline README.md:27]: 👍

File: .changeset/pink-owls-rhyme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+'davinci-github-actions': patch
+---
+
+---
+
+### danger
+- Use davinci in "package by package" way. Requires to install `@toptal/davinci-ci` in your project.
```
Comment (LashaJini) [inline .changeset/pink-owls-rhyme.md:8]: Nit

```suggestion
- Use davinci "package by package". Requires to install `@toptal/davinci-ci` in your project.
```

**Review by TomasSlama — APPROVED** [review]
👍

**Review by LashaJini — APPROVED** [review]
Looks gr8. Thanks Daniel.

**Review by dmaklygin — APPROVED** [review]
👍

File: .changeset/pink-owls-rhyme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+'davinci-github-actions': patch
```
Comment (OndrejTuma) [inline .changeset/pink-owls-rhyme.md:2]: Shouldnt this be treated as a breaking change? I assume that not everyone has all the parts installed so this update will require updates and testing

File: .changeset/pink-owls-rhyme.md (hunk 3 above)
Comment (denieler) [reply .changeset/pink-owls-rhyme.md:8]: @LashaJini IDK, to me it's less clear, maybe we can fix it in some other way?

File: .changeset/pink-owls-rhyme.md (hunk 4 above)
Comment (denieler) [reply .changeset/pink-owls-rhyme.md:2]: @OndrejTuma the functionality would not be broken even if you have `@toptal/davinci` installed

File: .changeset/pink-owls-rhyme.md (hunk 3 above)
Comment (LashaJini) [reply .changeset/pink-owls-rhyme.md:8]: It's understandable in both ways. So, np.

Comment (LashaJini) [reply .changeset/pink-owls-rhyme.md:8]: But maybe we know what we mean, by that.

Comment (LashaJini) [reply .changeset/pink-owls-rhyme.md:8]: Now that I think more about it, original one is better :+1:

File: .changeset/pink-owls-rhyme.md (hunk 4 above)
Comment (OndrejTuma) [reply .changeset/pink-owls-rhyme.md:2]: You're right, I didnt realize 😁 👍

**Review by OndrejTuma — APPROVED** [review]
Code looks good 💪 thanks

File: .changeset/pink-owls-rhyme.md (hunk 3 above)
Comment (LashaJini) [reply .changeset/pink-owls-rhyme.md:8]: Maybe, 

`- Use davinci by parts instead, in a "package by package" way. Requires...`

To emphasize the "package by package" part more

**Pull request merged**